src/AlphaHomoraV2/schema.graphql

```diff
@@ -0,0 +1,66 @@
+type AlphaHomoraV2Token @entity {
+  id: ID!
+  decimals: Int!
+  symbol: String!
+}
+
+type AlphaHomoraV2RewardToken @entity {
+  id: ID!
+  address: String!
+  token: AlphaHomoraV2Token!
+  lpToken: AlphaHomoraV2LpPair!
+}
+
+type AlphaHomoraV2LpPair @entity {
+  id: ID!
+  name: String!
+  token0: AlphaHomoraV2Token!
+  token1: AlphaHomoraV2Token!
+  exchange: String!
+  pid: Int!
+  wMasterChef: AlphaHomoraV2WMasterChef!
+  rewardTokens: [AlphaHomoraV2RewardToken!]! @derivedFrom(field: "lpToken")
+}
+
+type AlphaHomoraV2WMasterChef @entity {
+  id: ID!
+  key: String!
+  pairs: [AlphaHomoraV2LpPair!]! @derivedFrom(field: "wMasterChef")
+  chef: String!
+}
+
+type AlphaHomoraV2FarmData @entity {
```

Comment:
How can we get the TVL for a farm?

Reply:
TVL for farm data be obtained by multiplying the LP tokens locked value by the usd value of the lp token